style.css

```
@@ -0,0 +1,224 @@
+@import url('https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600&display=swap');
+
+*,
+*::before,
+*::after {
+  box-sizing: border-box;
+}
+
+html,
+body {
+  height: 100vh; 
+  margin: 0; 
+  font-family: 'Raleway', sans-serif;
+  background-color: #d2dcde;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.pricing-row {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  width: 100%;
+  max-width: 1200px;
+}
+
+.pricing-column {
+  flex: 1 1 248px;
+  background: #fff;
+  overflow: visible;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin: 1rem;
+  position: relative;
+}
+
+.pricing-column:nth-child(1) .pricing-header {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("image/ocean.png");
+  background-size: cover;
+  background-position: center;
+  
+}
+.pricing-column:nth-child(2) .pricing-header {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("image/mountain.png");
+  background-size: cover;
+  background-position: center;
+}
+.pricing-column:nth-child(3) .pricing-header {
+  background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)), url("image/forest.png");
+  background-size: cover;
+  background-position: center;
+}
```

```suggestion
.pricing-column .pricing-header {
  background-size: cover;
  background-position: center;
}

.pricing-column:nth-child(1) .pricing-header {
  background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)),
    url("image/ocean.png");
}

.pricing-column:nth-child(2) .pricing-header {
  background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)),
    url("image/mountain.png");
}

.pricing-column:nth-child(3) .pricing-header {
  background-image: linear-gradient(rgba(0, 0, 0, 0.4), rgba(0, 0, 0, 0.4)),
    url("image/forest.png");
}
```
sửa lại như này tránh lặp code